.github/workflows/create_release_pr.yml

```diff
@@ -0,0 +1,147 @@
+name: Release stage 1 - create release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version number (PEP 440 compliant e.g., 1.12.4, 5.0a2 etc)'
+        required: true
+      branch:
+        description: 'The branch to open the PR against'
+        required: false
+        default: 'master'
+
+env:
+  # Best not to include the GH token here, only do it for the steps that need it
+  PYPI_PACKAGE_NAME: 'metomi-isodatetime'
+  REPOSITORY: ${{ github.repository }}
+  INIT_FILE: 'metomi/isodatetime/__init__.py'
+  CHANGELOG_FILE: CHANGES.md
+  PR_BRANCH_PREFIX: 'prepare-'
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+
+    - name: Sanitise workflow inputs # Should be first step
+      shell: node {0}
+      env:
+        VERSION: ${{ github.event.inputs.version }}
+        BRANCH: ${{ github.event.inputs.branch }}
+      run: |
+        const versionStr = process.env.VERSION.trim();
+        console.log(`::set-env name=VERSION::${versionStr}`);
+        const branchStr = process.env.BRANCH.trim();
+        console.log(`::set-env name=BASE_REF::${branchStr}`);
+
+    - name: Checkout repo
+      uses: actions/checkout@v2.3.1
+      with:
+        ref: ${{ env.BASE_REF }}
+
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Configure git
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
What's `41898282`?

Reply:
I think this is just a random number which fits the GH email pattern.

Reply:
I took it from https://github.community/t/push-built-files-to-repo-using-actions/18186/2

Reply:
@MetRonnie found it in several other repositories (searching in GitHub). Thought it couldn't be a random number.

Then found that if you look at the Bot's profile: https://api.github.com/users/github-actions%5Bbot%5D

That's actually the Bot profile ID

![image](https://user-images.githubusercontent.com/304786/88905385-5628a900-d2aa-11ea-8e9a-89ebdea1957e.png)

Mystery solved!